packages/botonic-cli/src/commands/serve.ts

```diff
@@ -11,20 +16,122 @@ export default class Run extends Command {
     `$ botonic serve\n> Project is running at http://localhost:8080/`,
   ]
 
-  static flags = {}
+  static flags = {
+    preview: flags.boolean({
+      char: 'p',
+      description: 'Run preview Botonic 1.0 serve.',
+      default: false,
+    }),
+  }
 
   static args = []
 
   private telemetry = new Telemetry()
 
   async run(): Promise<void> {
     this.telemetry.trackServe()
-    this.parse(Run)
-    try {
-      console.log(colors.blue('\nServing Botonic...'))
-      spawnNpmScript('start')
-    } catch (e) {
-      console.log(e)
+    const { flags } = this.parse(Run)
+    if (flags.preview) {
+      const apps = ['websocket', 'rest', 'webchat', 'webviews']
+      const BASE_PATH = process.cwd()
+      const API_REST_DIR_SRC = path.resolve(BASE_PATH, './api')
+      const API_REST_PORT = 9010
+      const API_WS_DIR_SRC = path.resolve(BASE_PATH, './api')
+      const API_WS_PORT = 9091
+      const WEBCHAT_DIR_SRC = path.resolve(BASE_PATH, './webchat')
+      const WEBCHAT_PORT = 9000
+
+      const pgs = new PlaygroundService(API_REST_PORT)
+      try {
+        await pgs.start()
+      } catch (e) {
+        console.log("Couldn't start the Playground service: ", e)
+      }
```

Comment:
There's no need to add here the `try/catch` since the `start` method has already one inside and does not rethrow the exception if it fails.